src/controls/Controls.spec.js

```diff
@@ -62,4 +62,27 @@ describe('buttons text changes to reflect the state the door will be in if click
         expect(closed.textContent).not.toBe('Open');
         expect(locked.textContent).not.toBe('Unlock');
     })
+})
+
+describe('The toggle button is disabled', () => {
+    test('toggle is disabled if the gate is Open', () => {
```

Comment:
This is fine but you also need to check for lock as well as close gate. This allows you to verify that if you closed the gate you now have the gate locked. You can also check if you open the gate  that it changes the text to say that it can close it.